app/models/chapter.rb

```diff
@@ -34,5 +35,45 @@ def set_chapter_start_and_end_time_based_on_step_times
     self.update(start_time: starts, end_time: ends)
     self.save
   end
+
+ # Sorts through params in the controller to validate and save multiple steps for a chapter at once
+
+  def create_many_steps(params)
+    @new_steps = []
+    params.each do |key, value|
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.